### DISCUSSION TITLE
Integrate gauge with Expresso

### DISCUSSION THREAD

**Discussion by zbrea** [community/community#3012] [Other Feature Feedback, Questions, & Ideas] [answered]
Is it possible to integrate gauge (java) with Espresso framework for android application?
knowing that cucumber integrates well with espresso.

**Comment by evi-liu** [comment, marked as answer]
Hi there! This feedback forum is only for GitHub Discussions feedback. I'd recommend asking this questions on a forum more specific to your question.
